libs/core/tests/unit_tests/utils/test_function_calling.py

```diff
@@ -738,20 +736,17 @@ def test_tool_outputs() -> None:
     assert not response.tool_calls
 
 
-@pytest.mark.parametrize("use_extension_typed_dict", [True, False])
-@pytest.mark.parametrize("use_extension_annotated", [True, False])
+@pytest.mark.parametrize("typed_dict", [ExtensionsTypedDict, TypingTypedDict])
```

Comment:
(not important) -- any chance you know whether this will render nicely when looking at pytest output?

Reply:
Not very nicely, no.
I’ll add tags to make it pretty.